TODO

```diff
@@ -0,0 +1,5 @@
+# Known issues
+
+- Both the <job>.batch and the job.0, job.1, ... processes create the PBS_NODEFILE and change the jobs' environment.
```

Comment:
already open an issue with schedmd so they can have a look how to do this properly (ie what method to call) once we have a contract.

Reply:
Done.